product.html

```diff
@@ -0,0 +1,221 @@
+                <img src="./assets/icons/product/feat4.svg" alt="" class="prod-features__icon">
+                <div class="c-title text-2xl">Real time sync</div>
+                <div class="c-subtitle features_description">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.</div>
+            </div>
+
+            <div class="prod-features__item">
+                <img src="./assets/icons/product/feat5.svg" alt="" class="prod-features__icon">
+                <div class="c-title text-2xl">Built-in AI features</div>
+                <div class="c-subtitle features_description">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.</div>
+            </div>
+
+            <div class="prod-features__item">
+                <img src="./assets/icons/product/feat6.svg" alt="" class="prod-features__icon">
+                <div class="c-title text-2xl">Easy notes organization</div>
+                <div class="c-subtitle features_description">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.</div>
+            </div>
+        </div>
+    </section>
+
+    <section class="insights">
+        <img src="./assets/img/product/insigth-gradient.svg" alt="" class="insights__gradient">
+        <div class="insights__content">
+            <div class="insights__details">
+                <div class="insights__gradient-title">Marketing insights</div>
+                <div class="insights__title">Data-driven client feedback</div>
+                <div class="insights__subtitle">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.</div>
+            </div>
+            <div class="insights__details">
+                <img src="./assets/img/product/insight.png" alt="" class="insights__img">
+            </div>
+        </div>
+    </section>
+
+    <section class="insights">
+        <div class="insights__content insights__content--reverse">
+            <div class="insights__details">
+                <img src="./assets/img/product/ins-track.png" alt="" class="">
+            </div>
+            <div class="insights__details">
+                <div class="insights__gradient-title">Time tracker</div>
+                <div class="insights__title">Track your project performance</div>
+                <div class="insights__subtitle">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.</div>
+            </div>
+        </div>
+    </section>
+    
+    <section class="insights">
+        <img src="./assets/img/product/lead-ins-grad.svg" alt="" class="absolute right-0 -top-80 z-0">
+        <div class="insights__content relative z-10">
```

Comment:
You should add these classes to the CSS file and use a custom class name instead that follow BEM convention to be consistent.